js/alert.js

```diff
@@ -0,0 +1,26 @@
+const showError = ()=>{
+  const fragment = document.createDocumentFragment();
+  fragment.appendChild(document.querySelector('#error').content.cloneNode(true));
+  document.body.appendChild(fragment);
+
+  return new Promise ((response)=>{
+    document.querySelector('.error__button').addEventListener('click',()=>{
+      document.body.removeChild(document.querySelector('.error'));
+      response();
+    });
+  });
+};
+const showAlert = ()=>{
```

Comment:
Нужно добавить закрытие по esc. удаление этого ивента + закрытие по клику по подложке, вот пример
```suggestion
const showAlert = () => new Promise((response) => {
  const popup = document.querySelector('#success').content.cloneNode(true).querySelector('.success');

  const close = (e) => {
    if (e?.key && e.key !== 'Escape') {
      return;
    } else {
      e?.stopPropagation();
    }

    document.body.removeChild(document.querySelector('.success'));
    document.removeEventListener('keydown', close, {capture: true});
    response();
  };

  popup.querySelector('.success__button').addEventListener('click', close);
  popup.addEventListener('click', (e) => {
    if (e.target === popup) {
      close();
    }
  });

  document.addEventListener('keydown', close, {capture: true});
  document.body.appendChild(popup);
});
```